lib/puppet/provider/grafana_user/grafana.rb

```diff
@@ -48,79 +51,82 @@ def name
   end
 
   def name=(value)
-    resource[:name] = value
```

Comment:
These lines didn't actually do anything.  `resource[:foo]` was already `value`. (The function wouldn't have been called if the property was not set)